frontend/scripts/install_dev_env/install_linux.sh

```diff
@@ -49,12 +49,12 @@ else
     current_dir=$(pwd)
 
     cd $FLUTTER_PATH
-    # Use git to checkout version 3.7.5 of Flutter
-    git checkout 3.7.5
+    # Use git to checkout version 3.10.1 of Flutter
+    git checkout 3.10.1
     # Get back to current working directory
     cd "$current_dir"
 
-    echo "Switched to Flutter version 3.7.5"
+    echo "Switched to Flutter version 3.10.1"
 fi
```

Comment:
Refactor so that we only need to maintain this in one place, eg. `prepare_flutter.sh`